main/py-pbr/APKBUILD

```diff
@@ -7,24 +7,49 @@ pkgrel=0
 pkgdesc="Python Build Reasonableness"
 url="https://pypi.python.org/pypi/pbr"
 arch="noarch"
-license="APACHE2.0"
-depends="python2"
-depends_dev=""
-makedepends="$depends_dev python2-dev py-setuptools"
-install=""
-subpackages="$pkgname-dev"
+license="ASL 2.0"
```

Comment:
The [documentation](http://wiki.alpinelinux.org/wiki/Creating_an_Alpine_package#license) says otherwise. When I started to clean-up the license tags on a large scale a while back we agreed to follow the naming convention of Fedora/Arch Linux.